saleor/order/utils.py

```diff
@@ -944,8 +944,8 @@ def _update_order_total_charged(order: Order):
         sum(order.payments.values_list("captured_amount", flat=True))  # type: ignore
         or 0
     )
-    order.total_charged_amount += sum(
-        order.payment_transactions.values_list("charged_value", flat=True)  # type: ignore # noqa: E501
+    order.total_charged_amount += sum(  # type: ignore
```

Comment:
If I recall correctly this was failing when we upgraded mypy to the version that we currently have in repo, that's why it was moved to the line below.

Reply:
I'll test again but I had this failing in tests.

Reply:
Fixed.